### PR TITLE
Add keyless signing w/ storage in rekor to FUN.md

### PR DESCRIPTION
**Pull request by priyawadhwa**
I filled in the last section of this doc! Let me know if I didn't do this the intended way 😅

### CODE REVIEW

File: FUN.md [hunk 1]
```diff
@@ -60,4 +60,45 @@ Verified OK
 
 Store the signature in the Transparency Log and don't store the keys anywhere.
 
-**COMING SOON**
+```
```
Comment (squee1945) [inline FUN.md:63]: Do you want to adjust Hard Mode above to say that the public key is also stored in transparency log?

File: FUN.md [hunk 2]
```diff
@@ -60,4 +60,45 @@ Verified OK
...
+BtPsMIGNBggrBgEFBQcBAQSBgDB+MHwGCCsGAQUFBzAChnBodHRwOi8vcHJpdmF0
+ZWNhLWNvbnRlbnQtNjAzZmU3ZTctMDAwMC0yMjI3LWJmNzUtZjRmNWU4MGQyOTU0
+LnN0b3JhZ2UuZ29vZ2xlYXBpcy5jb20vY2EzNmExZTk2MjQyYjlmY2IxNDYvY2Eu
+Y3J0MCEGA1UdEQEB/wQXMBWBE3ByaXlhdzgxOUBnbWFpbC5jb20wCgYIKoZIzj0E
+AwMDaAAwZQIxAL2tZYELi3hGJS8sGTCGPKz83brdM4aak0v+SMMQGnyas7bY7tdm
+DoyAJmxwWx2ntQIwPyM1A5nE+f2Pg9CkQqyZEFS2sxQTdKBmQzODDn6GqVTJ7agN
+2djlcXFUJb1xFwO5
+-----END CERTIFICATE-----
+
+tlog entry created with index: 782549
```
Comment (squee1945) [inline FUN.md:89]: Related feedback: it seems like Cosign should emit an immutable https:// link to rekor for the tlog entry here.

File: FUN.md [hunk 3]
```diff
@@ -60,4 +60,45 @@ Verified OK
...
+MEUCIGi9CuxAceEcNkGLani2i3GdMgbl4bkGLILDhjh8n7DAAiEAhRwgfXMhnXyB38EZtOZX7fwtJBaSetMM88mZyYsV0pM=
+```
+
+Now find it from the log:
+
+```
+$ uuid=$(rekor-cli search --artifact <(git rev-parse HEAD) | tail -n 1)
```
Comment (squee1945) [inline FUN.md:96]: I assume `tail -n 1` retrieves the most recent signature for a given hash. I suppose a real world example would already know the git hash that it wanted to check...

Comment (priyawadhwa) [reply FUN.md:96]: yah we're just checking out the most recent signature here. realistically you might get a list & have to go through each entry to see if a trusted identity signed off on the commit.

**Pull request merged**